CHANGES.rst

```diff
@@ -176,6 +176,8 @@ tweakreg
 - Added the 'GAIADR3' catalog to the available options for alignment; 
   this has been enabled as the default option [#7611].
 
+- GAIADR3 catalog positions are corrrected for proper motions given an epoch [#7614]. 
```

Comment:
Since this PR did not make it into the 1.11.0 release, this entry will need to get moved up to the top (unreleased) section of the log.